Google & Unity Partnership Announcement Post (#91)
chromeos/chromeos.dev: pages/en/posts/google-unity-partner-to-support-chromebooks.md

---
title: Google & Unity partner to support Chromebooks
metadesc: Google is partnering with Unity to bring x86 build support and input and windowing improvements starting in Unity 2021.2.0a14.
tags:
  - product news
  - gaming
  - unity
  - keyboard support
  - mouse support

>   - large screen

Reviewer: ```suggestion
  - large screens
```
nit, but I think the tag is actually "large screens" plural

Author: Oh good catch!

  - window management
  - android
  - high-performance graphics
  - optimization
  - developer tools
authors:
  - mmonasch
featured:
  title: Google & Unity
  desc: We've partnered with Unity to support building games for Chromebooks
  images:
    - image: /images/posts/google-unity-partner-to-support-chromebooks/featured.png
      alt: Chrome OS gaming banner
hero:
  image: /images/posts/google-unity-partner-to-support-chromebooks/hero.png
  alt: Chrome OS gaming banner
  position: center
date: 2021-04-21
---

> Chrome OS users like to play Android games of all sorts, from card games to fully immersive open world RPGs. In the past year alone, time spent playing Android games on Chromebooks tripled. In order to bring the widest variety of games to our users, we enabled ARM games to run on Chrome OS [with Houdini](https://chromeos.dev/en/games/optimizing-games-publishing) and continue to invest heavily in Android gaming performance improvements. Our ultimate goal is to ensure that developers have the tools they need to build great games and empower them to bring incredible experiences to the platform.

Reviewer: ```suggestion
Chrome OS users like to play Android games of all sorts, from card games to fully immersive open world RPGs. In the past year alone, time spent playing Android games on Chromebooks tripled. In order to bring the widest variety of games to our users, we enabled ARM games to run on Chrome OS [with Houdini](/{{locale.code}}/games/optimizing-games-publishing) and continue to invest heavily in Android gaming performance improvements. Our ultimate goal is to ensure that developers have the tools they need to build great games and empower them to bring incredible experiences to the platform.
```

[Unity](https://unity.com/) is the most widely used game engine for developing Android games. To bring more popular and new games to our users, we’re excited to announce that Google has partnered with Unity to optimize their game engine for Chrome OS!

## An optimized engine for optimized games

Starting today with the alpha release of Unity 2021.2.0a14, any developer can begin building for Chrome OS. After selecting Android as the build target, developers can choose to also build for Chrome OS through the Player Settings. Allowing a Chrome OS artifact to be built from the same project and build target is convenient as it reduces duplicate code and streamlines the publishing pipeline. Additionally, by the end of 2021, the same features and toolset found in Unity's Android environment will be available for Chrome OS.

> #[Unity's Player Settings with x86 and x86_64 options](/images/posts/google-unity-partner-to-support-chromebooks/unity-x86-support.png)

Reviewer: ```suggestion
![Unity's Player Settings with x86 and x86_64 options](/images/posts/google-unity-partner-to-support-chromebooks/unity-x86-support.png)
```

Author: Huh! Weird that # did display the image. Going to change this locally and test since we don't have staging links currently and then I'll push changes back up. Thanks!
